internal/provider/cluster_resource_test.go

```diff
@@ -370,8 +387,8 @@ func serverlessClusterWithNoLimits(clusterName string) resource.TestStep {
 			resource.TestCheckNoResourceAttr(resourceName, "serverless.spend_limit"),
 			resource.TestCheckNoResourceAttr(resourceName, "serverless.usage_limits"),
 			resource.TestCheckResourceAttr(dataSourceName, "name", clusterName),
-			resource.TestCheckNoResourceAttr(dataSourceName, "serverless.spend_limit"),
```

Comment:
Why not assert the `spend_limit` field doesn't exist? Is the idea that we'll be getting rid of that field?

Reply:
Yeah, my thought was that it's just one less thing we'll have to change, but that's kind of a lame reason. I added the check back.